pkg/buildplan/buildplan.go

```diff
@@ -45,7 +68,7 @@ func Unmarshal(data []byte) (*BuildPlan, error) {
 }
 
 func (b *BuildPlan) Marshal() ([]byte, error) {
-	return json.Marshal(b.raw)
+	return json.MarshalIndent(b.raw, "", "  ")
```

Comment:
I don't think this has an effect on parsing, but MarshalIndent is for a more readable output. Was this for debugging and do we want to keep it?

Reply:
It was indeed for debugging, but I figured it would be worth keeping. It'll be useful if we're doing proxy debugging. And the extra whitespace really doesn't have a noticeable effect on performance.